spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -54,6 +56,10 @@ When("I type {string} into {string}", (value, element) => {
   get(element).type(value);
 });
 
+When("I type {string} into {string} and press enter", (value, element) => {
+  get(element).type(`${value}{enter}`);
+});
```

Comment:
For clearing, we have it as a separate step and I feel like it should be here too - so it'll be:

```
When I type "include" into "search box"
And I press enter in "search box"
```

By combining them in the same statement we end up with issues when we add new ones - e.g. if clearing was in the same statement, you'd have had to add two new rules for pressing enter, not one - and when another one is added in the future it'd require four new step definitions